bl-kernel/admin/controllers/edit-content.php

```diff
@@ -55,7 +55,7 @@
 		}
 	}
 
-	Redirect::page('content');
+	Redirect::page('content'.(isset($_POST['hash'])?$_POST['hash'].'/':''));
```

Comment:
I prefer to add here the hash # instead of send it in the variable.

Reply:
The hash # is already inside `window.location.hash`, so that would mean that you first have to remove it and then add it again. That's two extra operations.